### PR TITLE
✨ Add Orbit node

### PR DESCRIPTION
**Pull request by tanay1337**

### CODE REVIEW

File: packages/nodes-base/nodes/Orbit/ActivityDescription.ts [hunk 1]
```diff
@@ -0,0 +1,237 @@
...
+                description: 'A URL for the activity; displayed in the timeline',
+            },
+            {
+                displayName: 'Link Text',
+                name: 'linkText',
+                type: 'string',
+                default: '',
+                description: 'The text for the timeline link',
+            },
+            {
```
Comment (dzello) [inline packages/nodes-base/nodes/Orbit/ActivityDescription.ts:139]: We can remove the score field, this is something we are removing in the API soon.

File: packages/nodes-base/nodes/Orbit/MemberDescription.ts [hunk 2]
```diff
@@ -0,0 +1,380 @@
...
+            {
+                name: 'Lookup',
+                value: 'lookup',
+                description: 'Lookup a member or identity by GitHub username',
+            },
+            {
+                name: 'Update',
+                value: 'update',
+                description: 'Update a member',
+            },
```
Comment (dzello) [inline packages/nodes-base/nodes/Orbit/MemberDescription.ts:42]: There is a `find` method that we should include, it lets us find a member by one of their github/twitter/email/etc identities. It is now in the swagger docs at https://app.orbit.love/api-docs/index.html.

File: packages/nodes-base/nodes/Orbit/MemberDescription.ts [hunk 3]
```diff
@@ -0,0 +1,380 @@
...
+            {
+                displayName: 'Company',
+                name: 'company',
+                type: 'string',
+                default: '',
+                description: '',
+            },
+            {
+                displayName: 'Dev.to',
+                name: 'devTo',
```
Comment (dzello) [inline packages/nodes-base/nodes/Orbit/MemberDescription.ts:289]: We can remove devTo, this is now added as an identity.

File: packages/nodes-base/nodes/Orbit/Orbit.node.ts [hunk 4]
```diff
@@ -0,0 +1,326 @@
...
+import {
+	noteFields,
+	noteOperations,
+} from './NoteDescription';
+
+import {
+	postFields,
+	postOperations,
+} from './PostDescription';
+
```
Comment (dzello) [inline packages/nodes-base/nodes/Orbit/Orbit.node.ts:38]: I think we will need one more description, an `IdentityDescription`.

File: packages/nodes-base/nodes/Orbit/Orbit.node.ts [hunk 5]
```diff
@@ -0,0 +1,326 @@
...
+					if (additionalFields.description) {
+						body.description = additionalFields.description as string;
+					}
+					if (additionalFields.link) {
+						body.link = additionalFields.link as string;
+					}
+					if (additionalFields.linkText) {
+						body.link_text = additionalFields.linkText as string;
+					}
+					if (additionalFields.score) {
```
Comment (dzello) [inline packages/nodes-base/nodes/Orbit/Orbit.node.ts:150]: We can remove score here.

File: packages/nodes-base/nodes/Orbit/Orbit.node.ts [hunk 6]
```diff
@@ -0,0 +1,326 @@
...
+						body.bio = additionalFields.bio as string;
+					}
+					if (additionalFields.birthday) {
+						// This isn't functioning at the moment
+						body.birthday = additionalFields.birthday as string;
+					}
+					if (additionalFields.company) {
+						body.company = additionalFields.company as string;
+					}
+					if (additionalFields.devTo) {
```
Comment (dzello) [inline packages/nodes-base/nodes/Orbit/Orbit.node.ts:213]: We can remove devTo.

File: packages/nodes-base/nodes/Orbit/Orbit.node.ts [hunk 7]
```diff
@@ -0,0 +1,326 @@
...
+						body.devto = additionalFields.devTo as string;
+					}
+					if (additionalFields.linkedin) {
```
Comment (dzello) [inline packages/nodes-base/nodes/Orbit/Orbit.node.ts:216]: We can remove linkedin.

File: packages/nodes-base/nodes/Orbit/Orbit.node.ts [hunk 8]
```diff
@@ -0,0 +1,326 @@
...
+						body.linkedin = additionalFields.linkedin as string;
+					}
+					if (additionalFields.location) {
+						body.location = additionalFields.location as string;
+					}
+					if (additionalFields.name) {
+						body.name = additionalFields.name as string;
+					}
+					if (additionalFields.orbitLevel) {
```
Comment (dzello) [inline packages/nodes-base/nodes/Orbit/Orbit.node.ts:225]: Orbit level can be read but not written to. Do we still keep it in the definition?

**Comment by janober** [comment]
Thanks a lot. Got merged with #1123

**Comment by janober** [comment]
Got released with n8n@0.92.0

**Pull request closed without merging**